app/views/documents/show.html.erb

```diff
@@ -27,6 +27,14 @@
           <% end %>
         <% end %>
       <% end %>
+      <% if @document.class.has_organisations? %>
+        <dt>Primary Publishing Org</dt>
+        <dd><%= organisation_name(@document.primary_publishing_organisation) %></dd>
+        <dt>Organisations</dt>
+            <% @document.organisations.each do |org| %>
+            <dd><%= truncate(organisation_name(org), length: 140) %></dd>
```

Comment:
Wondered why we are truncating at that specific point but then realised it's the precedent for the other fields 👍